src/Knp/Rad/ResourceResolver/EventListener/ResourcesListener.php

```diff
@@ -54,6 +55,12 @@ public function resolveResources(FilterControllerEvent $event)
                 )
             ;
 
+            if ((!isset($resourceDetails['required']) || false !== $resourceDetails['required']) &&
```

Comment:
Why not merge `$resourceDetails` with default values, so you don't have to use `isset()` anymore.